### DISCUSSION TITLE
[Request] config option to recolor night vision UI

### DISCUSSION THREAD

**Discussion by JohnMetroid** [Malcolm-Q/LC-LateGameUpgrades#380] [General]
At present, the night vision UI clashes quite harshly with the rest of the game.  It would be ideal to allow recoloring via config

Thanks for the consideration.

**Comment by WhiteSpike** [comment]
So besides changing the lights color, you also want to change the UI that is displayed when using/recharging NV, yeah? Just to clarify what's being asked here.

**Comment by JohnMetroid** [comment]
Hi there,

Ideally there would be some means to: 1) Change the white UI elements associated with night vision to some other color (ideally the exact same orange-ish red as the rest of the UI elements for consistency) and 2) allow some method for recoloring the green energy bar

Thanks.

**Comment by WhiteSpike** [comment]
![image](https://github.com/Malcolm-Q/LC-LateGameUpgrades/assets/58040140/25129ad5-4a00-4634-a5c8-708ba414cd23)
```cfg


## The color used for the night vision's UI text.
# Setting type: Color
# Default value: FFFFFFFF
Night Vision UI Text Color = FF0000FF

## The color used for the night vision's UI battery bar.
# Setting type: Color
# Default value: 00FF00FF
Night Vision UI Bar Color = FFFF00FF
```

Thoughts? Is it close to what you wanted?

**Comment by JohnMetroid** [comment]
It looks great! Thanks for the update
